src/base/datetime/calendar/footer/style.ts

```diff
@@ -1,21 +1,26 @@
 export const CALENDAR_FOOTER_CSS = `
 kuc-base-datetime-calendar-footer,
 kuc-base-datetime-calendar-footer *,
-:lang(en) kuc-base-datetime-calendar-footer,
-:lang(en) kuc-base-datetime-calendar-footer * {
+kuc-base-datetime-calendar-footer:lang(en),
+kuc-base-datetime-calendar-footer:lang(en)* {
```

Comment:
I think it is incorrect 
>> kuc-base-datetime-calendar-footer:lang(en)*
kuc-base-datetime-calendar-footer:lang(en) *
has a space before * character